spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/security/BasicAuthSecurityConfiguration.java

```diff
@@ -123,4 +163,55 @@ protected void configure(HttpSecurity http) throws Exception {
 		http.sessionManagement().sessionCreationPolicy(SessionCreationPolicy.IF_REQUIRED);
 	}
 
+	/**
+	 * Read the configuration for "simple" (that is, not ACL based) security and apply it.
+	 */
+	private ExpressionUrlAuthorizationConfigurer<HttpSecurity>.ExpressionInterceptUrlRegistry configureSimpleSecurity(
+			ExpressionUrlAuthorizationConfigurer<HttpSecurity>.ExpressionInterceptUrlRegistry security) {
+		for (String rule : authorizationConfig.getRules()) {
+			Matcher matcher = AUTHORIZATION_RULE.matcher(rule);
+			Assert.isTrue(matcher.matches(),
+					String.format(
+							"Unable to parse security rule [%s], expected format is 'HTTP_METHOD ANT_PATTERN => SECURITY_ATTRIBUTE(S)'",
+							rule));
+
+			HttpMethod method = HttpMethod.valueOf(matcher.group(1).trim());
+			String urlPattern = matcher.group(2).trim();
+			String attribute = matcher.group(3).trim();
+
+			logger.error(">>>>Authorization '{}' | '{}' | '{}'", method, attribute, urlPattern);
+			security = security.antMatchers(method, urlPattern).access(attribute);
```

Comment:
User with `ROLE_ADMIN` privilege can't access the URL mapped under `ROLE_CREATE`. For instance, user with `ROLE_ADMIN` can't register apps as the registration POST URL is mapped for `ROLE_CREATE`.